bench/bench.py

```diff
@@ -76,20 +79,17 @@ def _bootstrap(addr):
             'sudo -S tar -C /usr/local -xzf go%s.linux-amd64.tar.gz' % golang_version,
             'sudo -S apt-get update',
             'sudo -S apt-get -y install git mercurial',
+			'sudo -S apt-get -y install golang',
+			'curl https://raw.githubusercontent.com/golang/dep/master/install.sh > installdep.sh',
+			'chmod +x installdep.sh',
+			'sudo GOPATH=/usr/local/go ./installdep.sh',
             'mkdir -p go/src/github.com/nsqio',
             'cd go/src/github.com/nsqio && git clone https://github.com/nsqio/nsq',
             'cd go/src/github.com/nsqio/nsq && git checkout %s' % commit,
-            'sudo -S curl -s -o /usr/local/bin/gpm \
-                https://raw.githubusercontent.com/pote/gpm/v1.2.3/bin/gpm',
-            'sudo -S chmod +x /usr/local/bin/gpm',
-            'cd go/src/github.com/nsqio/nsq && \
-                GOPATH=/home/ubuntu/go PATH=$PATH:/usr/local/go/bin gpm install',
-            'cd go/src/github.com/nsqio/nsq/apps/nsqd && \
-                GOPATH=/home/ubuntu/go /usr/local/go/bin/go build',
-            'cd go/src/github.com/nsqio/nsq/bench/bench_writer && \
-                GOPATH=/home/ubuntu/go /usr/local/go/bin/go build',
-            'cd go/src/github.com/nsqio/nsq/bench/bench_reader && \
-                GOPATH=/home/ubuntu/go /usr/local/go/bin/go build',
+			'cd go/src/github.com/nsqio/nsq && GOPATH=/home/ubuntu/go /usr/local/go/bin/dep ensure',
+			'cd go/src/github.com/nsqio/nsq && make',
+			'cd go/src/github.com/nsqio/nsq/bench/bench_writer && GOPATH=/home/ubuntu/go /usr/local/go/bin/go build',
+            'cd go/src/github.com/nsqio/nsq/bench/bench_reader && GOPATH=/home/ubuntu/go /usr/local/go/bin/go build',
```

Comment:
The inconsistent indenting shown here is because you're using tabs for some lines. Because this is python, 4-space indent everywhere is the way to go.